test/test_other.py

```diff
@@ -3083,14 +3083,14 @@ def test_embind_tsgen_ignore(self):
                   '-lembind', # Test duplicated link option.
                   ]
     self.emcc(test_file('other/embind_tsgen.cpp'), extra_args)
-    self.assertFileContents(test_file('other/embind_tsgen.d.ts'), read_file('embind_tsgen.d.ts'))
+    self.assertFileContents(test_file('other/embind_tsgen_ignore_1.d.ts'), read_file('embind_tsgen.d.ts'))
```

Comment:
What rename these?  How do they differ?

Reply:
They have different options enabled which changes the output of the wasm module exports.